tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, pypy, py33, py34, py35, py27-flake8, py35-flake8, integration
+envlist = py27, pypy, py33, py34, py35, py27-flake8, py36-flake8, integration
```

Comment:
`pypy3` doesn't need to appear here?

Reply:
envlist is used when `TOXENV` isn't set, which only happens when testing locally.

It should be here to make the local test list similar to travis.